README.md

```diff
@@ -1,32 +1,55 @@
-# Project name
+# Nucmorph-Colorizer 🔬🎨
 
-One line description
-
----
+#### A web visualizer for time-sequence data from the Nuclear Morphogenesis project.
 
 ## Description
 
-Main features in a brief descriptive text.
+Nucmorph-Colorizer is a web tool for interacting with data from the nuclear morphogenesis project! You can apply color maps and ranges,
+switch between features, play frames to observe motion, and view plots showing how features change over time.
+
+![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/d9d22cba-faa0-4366-a647-973bc2fce360)
 
 ## Installation
 
-Describe how to obtain the software and get it ready to run
+**Prerequisites:**
+
+- Node
+- Python 3 (and optionally, a virtual Python environment)
+
+Clone this git repository.
+
+### Web tool
 
-## Documentation
+Navigate to the project's root directory in a terminal window.
 
-If you have more extensive technical documentation (whether generated or not), ensure they are published to the following address:
-For full package documentation please visit
-[organization.github.io/projectname](https://organization.github.io/projectname/index.html).
+```
+npm install
+npm run dev
+```
 
-## Quick Start
+This will start a development server you can access from your browser. By default, the server will be hosted at `http://localhost:5173/`.
 
-To generate some test data,
+### Data preprocessing
 
-cd scripts
+Data must be preprocessed to work with Nucmorph-Colorizer. [You can read more about the data format specification here.](./scripts/timelapse-colorizer-data/generate_data.py) Utilities are included in this repository to convert standard time-series data.
+
+For loading of datasets to work correctly, you'll need to run these commands from a device that has access to Allen Institute's on-premises data storage. If running off of shared resources, remember to initialize your virtual environment first! This may look like `conda activate {my_env}`.
+
+Navigate to the project's scripts directory.
+
+```
+cd scripts/
 pip install .
-python generate_test_data.py
+```
+
+The `generate_data.py` script can take in a named dataset (like `baby_bear`, `mama_bear`, or `goldilocks`) and convert it to a format readable
+by the web client.
+
+```
+python timelapse-colorizer-data/generate_data.py --output_dir {output_dir} --dataset {dataset_name}
+```
```

Comment:
there is more to say here about what to do with the data that lands in output_dir because we only have one location where the data is currently searched for in on-prem storage.  Maybe could copy/paste some of what's in generate_data.py or just point people to instructions in that file.

Reply:
Added a comment about this, with a reference to a future feature improvement.